okta/data_source_okta_group.go

```diff
@@ -18,6 +18,13 @@ func dataSourceGroup() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"type": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      false,
```

Comment:
🤔 I'm trying to read up on where that comes from - can you give me an article or documentation reference to look at?

Reply:
I think this has something to do with how the terraform converts values from the interface into the value. Because type is `schema.TypeString`, I think it just represents false as 0 (because false internally is just integer) and then converts it into the string (using `strings.Itoa` I guess), which is "0".